### PR TITLE
Fix in context translation

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
In context translation is not currently working due to:
* Missing font files for the made up language
* Reliance on the supported languages json for language info

This PR fixes that by:
* Updating the font generation code to fix a capitalization error (`bold` instead of `Bold` and `regular` instead of `Regular`)
* Updating the font generation code to include the made up locale for in context translation
* Rerunning the font generation code
* Modifying the frontend code to use the dynamically generated language info bootstrapped into the frontend by Django, as opposed to the static supported languages json

### Reviewer guidance
Download the pex file.
Run the pex with this command:
`DJANGO_SETTINGS_MODULE=kolibri.deployment.default.settings.translation ./<pex_file> start`
Test that the in context translations display properly!

----

### Contributor Checklist

- [ ] Contributor has fully tested the PR manually
- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: build_tools/i18n/fonts.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def _ttf_font_path(font_info, is_ui=False, is_bold=False):
     will fit within UI elements.
     """
     font_name = font_info["name"]
-    weight = "bold" if is_bold else "regular"
+    weight = "Bold" if is_bold else "Regular"
```
Comment (indirectlylit) [inline build_tools/i18n/fonts.py:128]: whoa! wondering how this slipped by me ...

Comment (indirectlylit) [reply build_tools/i18n/fonts.py:128]: Oh, paths are case-insensitive on Mac.

**Review by indirectlylit — APPROVED** [review]

**Comment by radinamatic** [comment]
TIL in-context translation can be run locally! :relaxed: 

So technically, translators could run Kolibri with these settings and translate on their own device, and the changes they make get immediately uploaded directly to Crowdin? Awesome :100:

**Comment by radinamatic** [comment]
Could this be run on Windows? I wasn't able to run it even after setting `DJANGO_SETTINGS_MODULE` as an environment variable with the value of `kolibri.deployment.default.settings.translation`:

**Comment by radinamatic** [comment]
My bad, seems to be working on Windows too! :+1:

**Pull request merged**